oauth2-authorization-server/spring-security-oauth2-authorization-server.gradle

```diff
@@ -8,6 +8,7 @@ dependencies {
 	compile springCoreDependency
 	compile 'com.nimbusds:nimbus-jose-jwt'
 	compile 'com.fasterxml.jackson.core:jackson-databind'
+	compile 'net.minidev:json-smart'
```

Comment:
Remove `net.minidev:json-smart` since this will be transitively imported from Spring Security.